packages/openapi-typescript/package.json

```diff
@@ -6,6 +6,12 @@
     "name": "Drew Powers",
     "email": "drew@pow.rs"
   },
+  "contributors": [
```

Comment:
We’re not using this field in `package.json`. We’re tracking contributors via the docs site. Let’s remove it.